cmd/buildkitd/main_containerd_worker.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux || windows || freebsd
-// +build linux windows freebsd
+//go:build linux || windows || freebsd || darwin
+// +build linux windows freebsd darwin
```

Comment:
Maybe just remove build tags here? If someone ports buildkit, they already have containerd, so have zero trouble supporting this.